controllers/loadtest_controller.go

```diff
@@ -125,6 +124,69 @@ func (r *LoadTestReconciler) Reconcile(req ctrl.Request) (ctrl.Result, error) {
 	return ctrl.Result{}, nil
 }
 
+// checkMissingPods attempts to check if any pods is missing for current
+// loadtest.
```

Comment:
nit: @paulosjca may chime in, since he's good with documentation... but I would consider elaborating a little more about what it accepts and what is returned. Otherwise, good to me 👍🏻

Reply:
Just a small thing, when using 'load test' as plain English, I would use two words (this may be true elsewhere as well). Also, 'the current load test'.

Reply:
I will modify the loadtest to load test, that is a great suggestion! Also try to explain the method clearer.